src/reactThisBindingIssueRule.ts

```diff
@@ -53,6 +71,17 @@ class ReactThisBindingIssueRuleWalker extends ErrorTolerantWalker {
         this.getOptions().forEach((opt: any) => {
             if (typeof(opt) === 'object') {
                 this.allowAnonymousListeners = opt['allow-anonymous-listeners'] === true;
+                if (opt['bind-decorators']) {
+                    const allowedDecorators: any[] = opt['bind-decorators'];
+                    if (
+                        !Array.isArray(allowedDecorators.constructor)
```

Comment:
Check should be for `allowedDecorators` not it's constructor.

Error below can be rephrase to mention that `bind-decorators` should be a list of strings, because now it will fail if string is passed, and not an array

Reply:
ahh, I missed the allowedDecorators.constructor part for some reason, my bad, will fix after I wake up now.